github/github.go

```diff
@@ -3,6 +3,16 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// go-github authors should run the following commands before
+// sending PRs to GitHub:
+//
+//     cd github.com/go-github
+//     go generate ./...
+//     go test ./...
+//     go vet ./...
```

Comment:
How about deleting the `cd` step and making these commands use absolute import paths rather than relative? E.g.:

```
go generate github.com/google/go-github/...
```

Then the same command can be run from any directory, and it'll do the right thing.